experimental/origin-dapp2/src/pages/growth/Invite.js

```diff
@@ -0,0 +1,252 @@
+import React, { Component, Fragment } from 'react'
+import Link from 'components/Link'
+
+function NavigationItem(props) {
+  const { selected, onClick, title } = props
+  return (
+    <a
+      href="#"
+      onClick={e => {
+        e.preventDefault()
+        onClick()
+      }}
+      className="pt-4 pb-43 pr-4"
+    >
+      <div className="d-flex flex-column align-items-center">
+        <div className={`title ${selected ? 'active' : ''}`}>{title}</div>
+        {selected && <div className="select-bar" />}
+      </div>
+    </a>
+  )
+}
+
+class GrowthInvite extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      subPage: 'sendInvites',
+      inviteCode: 'origin-invite-code',
+      showCopyConfirmation: false
+    }
+
+    this.inviteCode = `${location.protocol}//${location.hostname}/#/welcome/${
+      this.state.inviteCode
+    }`
+
+    this.handleNavigationClick = this.handleNavigationClick.bind(this)
```

Comment:
The way you're calling these methods, ie inside a `onClick={() => this.fn()}`, I don't think you need to bind them any more. The rest of dapp2 doesn't have binding in constructors. Not a big deal but thought I would mention :-)

Reply:
Good point, removing the `bind` and changing to current DApp2 style.